hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/NSSummaryTask.java

```diff
@@ -137,17 +151,23 @@ public Pair<String, Boolean> reprocess(OMMetadataManager omMetadataManager) {
         }
       }
     } catch (InterruptedException ex) {
-      LOG.error("Error while reprocessing NSSummary " +
-          "table in Recon DB. ", ex);
+      LOG.error("Error while reprocessing NSSummary table in Recon DB.", ex);
       return new ImmutablePair<>(getTaskName(), false);
     } catch (ExecutionException ex2) {
-      LOG.error("Error while reprocessing NSSummary " +
-          "table in Recon DB. ", ex2);
+      LOG.error("Error while reprocessing NSSummary table in Recon DB.", ex2);
       return new ImmutablePair<>(getTaskName(), false);
     } finally {
       executorService.shutdown();
+
+      long endTime = System.nanoTime(); // Record end time
+      long durationInMillis = (endTime - startTime) / 1_000_000; // Convert to milliseconds
```

Comment:
Use `TimeUnit.NANOSECONDS.toMillis()` for conversion.

Reply:
Thank you for the suggestion. I have implemented the recommended changes.